src/nvim/if_cscope.c

```diff
@@ -1509,7 +1532,8 @@ static char *cs_parse_results(int cnumber, char *buf, int bufsize, char **contex
  */
 static void cs_file_results(FILE *f, int *nummatches_a)
 {
-  int i, j;
+  size_t i;
+  int j;
```

Comment:
My first thought was that you could see all the variables used in the function by just looking at the top of the function. But I think I've changed my mind, and I agree that it is more readable if defined inline in the loop declaration in this case. Because declaring the variable inside the loop declaration also limits the variable's scope, so that you know that it doesn't remain relevant beyond the loop.